.github/workflows/scripts/emscripten-build.sh

```diff
@@ -11,7 +11,7 @@ emmake make &&
 # install to obtain a table folder which does not contain build scripts
 emmake make install prefix="$(pwd)/out-emscripten-install"
 
-buildjs "16" "build-no-tables-utf16.js" &&
+#buildjs "16" "build-no-tables-utf16.js" &&
```

Comment:
I don't understand why this is commented out

Reply:
We were not publishing the 16 bit version (and we don't need to), so it was no use building it.